package.json

```diff
@@ -107,7 +109,6 @@
     ],
     "nohoist": [
       "**/@types/*",
-      "**/@types/*/**",
```

Comment:
What's going on here?

Reply:
the issue is that `@types/graphql-upload`  has `graphql` as a dependency https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/graphql-upload/package.json#L4 and that makes `apollo-server-core` upset, because it expects to have only one `graphql` instance in the project, but because we have `"**/@types/*/**"`,  in `hoist` it installs `@types/graphql-upload` and `graphql` inside `apollo-server-core`